documentation/API-Readiness-Checklist.md

```diff
@@ -0,0 +1,26 @@
+# API Readiness Checklist
+
+Checklist for api-name api-version in rx.y
+
+| Nr | API release assets  | alpha | release-candidate |  public-release<br>initial | public-release<br> stable | Status | Comments |
+|----|----------------------------------------------|:-----:|:-----------------:|:-------:|:------:|:----:|:----:|
+|  1 | API definition                               |   M   |         M         |    M    |    M   |      |      |
+|  2 | Design guidelines from Commonalities applied |   O   |         M         |    M    |    M   |      |      |
+|  3 | Guidelines from ICM applied                  |   O   |         M         |    M    |    M   |      |      |
+|  4 | API versioning convention applied            |   M   |         M         |    M    |    M   |      |      |
+|  5 | API documentation                            |   M   |         M         |    M    |    M   |      |      |
+|  6 | User stories                                 |   O   |         O         |    O    |    M   |      |      |
+|  7 | Basic API test cases & documentation         |   O   |         M         |    M    |    M   |      |      |
+|  8 | Enhanced API test cases & documentation      |   O   |         O         |    O    |    M   |      |      |
+|  9 | Test result statement                        |   O   |         O         |    O    |    M   |      |      |
+| 10 | API release numbering convention applied     |   M   |         M         |    M    |    M   |      |      |
+| 11 | Change log updated                           |   M   |         M         |    M    |    M   |      |      |
+| 12 | Previous public-release was certified        |   O   |         O         |    O    |    M   |      |      |
```

Comment:
If enhanced test plan is not required for the initial public release, and certification of the initial public release is required to have a stable public-release, this is assuming that the test plan used for the first certification is just the basic one, not the enhanced one, is this right? Or is that the initial public release has to be certified once the complete test plan is available.